extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -352,4 +357,20 @@ export class AmpAd3PImpl extends AMP.BaseElement {
     }
     return true;
   }
+
+  addDefaultFallback() {
+    if (this.placeholder_) {
+      this.togglePlaceholder(false);
+    }
+    const defaultFallback = this.win.document.createElement('div');
+    setStyles(defaultFallback, {
```

Comment:
Nah, we're already inside a mutate context (from the [here](https://github.com/ampproject/amphtml/pull/4248/files#diff-f9622dc64dc2dbb6729b43a96a27c1a0R316)). Though, I'd move that `#defereMutate` into this function to make it more obvious.